release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2306,6 +2314,8 @@ Workaround: Use OpenShift OAuth for authentication. (link:https://bugzilla.redha
 
 * Previously, the `oc adm must-gather` tool did not collect performance specific data when more than one `--image` argument was supplied. Files, including node and performance related files, were missing when the operation finished. The issue affects {product-title} versions between 4.7 and 4.10. This issue can be resolved by executing the `oc adm must-gather` operation twice, once for each image. As a result, all expected files can be collected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2018159[*BZ#2018159*])
 
+* This release contains a known issue with the Operator SDK scorecard storage utility. Currently, running the `operator-sdk scorecard` command with the `--test-output` flag returns an empty directory. There is no known workaround at this time. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2017669[*BZ#2017669*])
```

Comment:
From the other PR, it sounds like specifying nothing gets you into a better place. I suggest mentioning that omitting the flag stores the content in the `test-output` directory of your working directory here, too.